xblock/field_data.py

```diff
@@ -154,8 +156,24 @@ def _field_data(self, block, name):
 
         return self._scope_mappings[scope]
 
-    def get(self, block, name):
-        return self._field_data(block, name).get(block, name)
+    def _shared_field_data(self, remote_field_scope):
+        """Summary
+        
+        Args:
+            block (TYPE): Description
+            name (TYPE): Description
+        
+        Returns:
+            TYPE: Description
+        """
+        return self._scope_mappings[remote_field_scope]
+
+
+    def get(self, block, name, remote_field_scope=None):
+        if isinstance(block, XBlock):
+            return self._field_data(block, name).get(block, name)
+        elif isinstance(block, ScopeIds):
+            return self._shared_field_data(remote_field_scope).get(block, name)
```

Comment:
I need the ScopeIds so that I can query on user id, while xblock is still necessary because I need to access the attaching field data. But let me review the workflow to ensure it is working as expected. 

Thank you!